src/servers/http_server.cc

```diff
@@ -574,9 +627,25 @@ HTTPServer::Create(
     std::string addr = "0.0.0.0:" + std::to_string(ep_map.first);
     LOG_INFO << "Starting HTTPService at " << addr;
     http_servers->emplace_back(
-        new HTTPServerImpl(server, ep_map.second, ep_map.first, thread_cnt));
+        new HTTPAPIServer(server, ep_map.second, ep_map.first, thread_cnt));
   }
 
   return Status::Success;
 }
+
+Status
```

Comment:
Can we change Create() to CreateAPIServer() to keep the function names consistent.